<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KiertoMaaritys</class>
 <widget class="QWidget" name="KiertoMaaritys">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1,0">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Laskujen kierto</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Määritellyt kierrot</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListView" name="view"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="uusiNappi">
          <property name="text">
           <string>Uusi</string>
          </property>
          <property name="icon">
           <iconset resource="../pic/pic.qrc">
            <normaloff>:/pic/lisaa.png</normaloff>:/pic/lisaa.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="muokkaaNappi">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Muokkaa</string>
          </property>
          <property name="icon">
           <iconset resource="../pic/pic.qrc">
            <normaloff>:/pic/muokkaa.png</normaloff>:/pic/muokkaa.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="poistaNappi">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Poista</string>
          </property>
          <property name="icon">
           <iconset resource="../pic/pic.qrc">
            <normaloff>:/pic/poista.png</normaloff>:/pic/poista.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="portaaliRyhma">
     <property name="title">
      <string>Laskutusportaali</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="eiPortaalissaLabel">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 0);</string>
        </property>
        <property name="text">
         <string>Yhtään kiertoa ei ole määritelty näkyväksi portaalissa.</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Osoite</string>
          </property>
          <property name="buddy">
           <cstring>osoiteEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="osoiteEdit">
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Oma
ohje</string>
          </property>
          <property name="buddy">
           <cstring>alatunnisteEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPlainTextEdit" name="alatunnisteEdit"/>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="ocrCheck">
          <property name="text">
           <string>Käytä tekstintunnistusta</string>
          </property>
          <property name="icon">
           <iconset resource="../pic/pic.qrc">
            <normaloff>:/pic/teksti.png</normaloff>:/pic/teksti.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="tallennaNappi">
          <property name="text">
           <string>Tallenna</string>
          </property>
          <property name="icon">
           <iconset resource="../pic/pic.qrc">
            <normaloff>:/pic/ok.png</normaloff>:/pic/ok.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PeruNappi">
          <property name="text">
           <string>Peru</string>
          </property>
          <property name="icon">
           <iconset resource="../pic/pic.qrc">
            <normaloff>:/pic/peru.png</normaloff>:/pic/peru.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../pic/pic.qrc"/>
 </resources>
 <connections/>
</ui>
